samples/boards/cc3220sf_launchxl/echo/src/simplelink_support.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2017, Texas Instruments Incorporated
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+/* This method of passing SSID and Password on the command line is in lieu
+ * of a more powerful, complex method of device provisioning.
+ * TI SimpleLink supports a few options for provisioning, the most common
+ * in use being AP mode provisioning for IoT headless devices.
+ * This should be added in the future; until then, at least this manual
+ * method avoids storing one's WiFi password into a Kconfig file.
+ * To build:
+ *  % export CFLAGS="-DNET_SSID="<SSID_Name>" -DNET_PASS="<password>""
+ *  % mkdir build && cd build
+ *  % cmake .. -DBOARD=cc3220sf_launchxl
+ *  % make
+ */
+#ifdef NET_SSID
+#define SSID_NAME       STRINGIFY(NET_SSID)
+#else
+#define SSID_NAME	"DummySSID"
```

Comment:
Should this raise an error if NET_SSID or NET_PASS is not defined?

Reply:
Acutally, that would cause a build test via CI script (or shippable) to fail.  The main requirement is that SSID_NAME and SECURITY_KEY are defined.  NET_SSID and NET_SECURITY are only used here to get the correct keys into the build for a run-time test, and we don't want to store any real passwords in any source file.